CHANGELOG.md

```diff
@@ -1,112 +1,24 @@
-* **[Issue 137](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/137):** Not adding '?' delimiter if the endpoint URL already has one ([PR 207](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207)).
-* **[Issue 149](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/149):** Renaming the tests folder to "test" so that build tools can find and run the tests ([PR 148](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/148)).
-* **[Issue 174](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/174):** Calling IssuerSigningKeyValidator delegate if set by the user ([PR 207](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207)).
-* **[Issue 176](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/176):** Adding `OpenIdConnectConfiguration.Write` method to serialize `OpenIdConnectConfiguration` object back to JSON ([PR 218](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/218)).
-* **[Issue 183](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/183):** Fixing the "double" await causing a deadlock in `HttpDocumentRetriever.cs` ([PR 207](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207)).
-* **[Issue 201](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/201):** Adding comments to `TokenValidationParameters` to explain what each property intends to do and possible security implications of turning off the default validation ([PR 217](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217)).
-
-
 ## Features
 * **JsonWebKey:** Upgraded to fully support features in http://tools.ietf.org/html/draft-ietf-jose-json-web-key-37
 * **Logging:** Added Event Source based logging in Wilson. `IdentityModelEventSource` is the exposed event source handle.
-* The OM removed support for `SecurityTokens` for crypto operations
-
-## Breaking Changes
+* **Cryptography:** Added support for Elliptical Curve (ECDsa) algorithm.
 
-* **NOTE:** We have not added support for WsFederation and Saml in this version yet.
+## Major Changes
+* Dropped support for WsFederation and Saml.
+* `SecurityKey` is used for token validation instead of `SecurityToken`. 
+* `CryptoProviderFactory` provides all the extensibility to control and customize crypto support.
+* Dropped support for reading and writing JWT tokens embedded in XML.
+* Removed dependency on `System.IdentityModel.dll`.
+* Replaced `JavaScriptSerializer` by Json.Net serializer. Deserializing `int` results in an `Int64` object by default instead of `Int32`.
```

Comment:
"by" should be replaced _by_ "with" here. Source: http://english.stackexchange.com/a/151352 :smile: